demo/seed-devnet.sh

```diff
@@ -43,7 +43,7 @@ function hnode() {
   else
       docker run --rm -it \
         -v ${SCRIPT_DIR}/devnet:/devnet \
-        ghcr.io/input-output-hk/hydra-node -- ${@}
+        ghcr.io/input-output-hk/hydra-node:unstable -- ${@}
```

Comment:
I think we should prioritize maintaining the stable version (latest release) instead.

Imo, would be enough to add a disclaimet header to the file specifying the version it supports.

Another option would be to maintain 2 version of scripts: one for stable and another for unstable.

Wdyt?

Reply:
I'm strongly against maintaining two versions.

I'm with Franco on the fact that we should encourage use of stable versions. In fact, this will work as soon as we did a release (and the implicit `:latest` docker tag will be used here), right?

Reply:
What I noticed is the `latest` tag doesn't point to the last release. For example [here](https://github.com/input-output-hk/hydra/pkgs/container/hydra-node) `latest` points to `0.9.0` when we had a follow up release of `0.9.1`. Why don't we use the last release tag and make sure to add notes to our release process to also update the `seed-devnet` script to use appropriate tag? cc @ch1bo

Reply:
Tag 0.9.0 is _doomed_ to have this problem when we release 0.10.0 of being broken. Let's make our piece with that but I'd like to not have the same issue with 0.10.0.